src/pages/documentation/edge-network/release-notes.md

```diff
@@ -8,6 +8,15 @@ keywords:
 
 # Release Notes
 
+## September 1, 2023
+
+### iOS Edge 4.2.0
+* Added the ability to request a "completion" event when making an experience event request. The `contentComplete` event is dispatched after the request connection is closed.
```

Comment:
Missing a newline between the header and the bullet point. (This will cause it to fail validation.)